Formula/proper.rb

```diff
@@ -0,0 +1,23 @@
+class Proper < Formula
+  desc "PropEr: a QuickCheck-inspired property-based testing tool for Erlang"
+  homepage "https://proper-testing.github.io"
+  url "https://github.com/proper-testing/proper/archive/v1.3.tar.gz"
+  sha256 "7e59eeaef12c07b1e42b0891238052cd05cbead58096efdffa3413b602cd8939"
+
+  depends_on "erlang"
+
+  def install
+    system "make"
+    prefix.install Dir["ebin", "include"]
+  end
+
+  def caveats; <<~EOS
+    Add the following line to your shell startup file (~/.bashrc in the case of the Bash shell):
+      export ERL_LIBS=#{opt_prefix}
```

Comment:
We don't typically include this for other Erlang formulas, so I suggest to remove it.

Reply:
@fxcoudert The `caveats` method itself that is?

Reply:
Yes

Reply:
@fxcoudert Then what's the way to communicate the necessary post-install steps?

Reply:
In `caveats` if they are specific to Homebrew. Otherwise, the software's documentation should be where this information is found.

Reply:
@fxcoudert Got it. I've dropped the `caveats` method.